imports/ui/HongKongRanking.html

```diff
@@ -2,16 +2,16 @@
 	<h2>Hong Kong Mahjong ELO Rankings</h2>
 	<table>
 		<tr>
-			<th align=center width=200px>Players</th><th width=50px></th><th>ELO</th>
+			<th align=center width=50px>Rank</th>
```

Comment:
Ignore the fact you just copied me, I think we should commit this and wait for frontend to fix